lib/saluki-core/src/data_model/event/log/mod.rs

```diff
@@ -0,0 +1,85 @@
+//! Logs.
+
+use std::num::NonZeroU64;
+
+use saluki_context::tags::SharedTagSet;
+use stringtheory::MetaString;
+
+/// Internal representation of a log event aligned to Datadog logs payload fields https://docs.datadoghq.com/api/latest/logs/#log-object.
```

Comment:
We should just say "A log event." here.

In general, we try to avoid language about exactly how one field maps to the equivalent field in the Agent codebase, or some specific Datadog feature, and so on. At the end of the day, yeah, we're using Saluki to build Agent Data Plane, but there's no reason that it couldn't be used to build something generic, and that's the mindset we want to be in when writing this stuff.

The exception here is when a field is truly Datadog-specific, in terms of the format of the data itself or something along those lines.